app/mailers/notifier.rb

```diff
@@ -76,6 +77,15 @@ def order_detail_status_change(order_detail, old_status, new_status, to)
 
   private
 
+  def attach_ical(order)
+    order.order_details.map(&:reservation).compact.each do |reservation|
+      calendar = ReservationCalendar.new(reservation)
+      attachments[calendar.filename] = {
```

Comment:
What I was really saying is does gmail recognize it as an event invitation? If we can't test this until it gets to stage, that's fine, we should be able to verify from open stage.